recipes/i2c-tools/all/conanfile.py

```diff
@@ -0,0 +1,43 @@
+import os
+from conans import ConanFile, CMake, tools, AutoToolsBuildEnvironment
+
+
+class I2cConan(ConanFile):
+    name = "i2c-tools"
+    license = "LGPL 2.1"
+    author = "Eirik Brandtzæg eirikb@eirikb.no"
```

Comment:
cci does not add authors to the recipes.
We assume cci is the author.
```suggestion
```

Reply:
Thanks. I was really wondering about this one.
I'm not the author of the library either.
Just used the defaults from conan new